Source/Core/TerrainCompression.js

```diff
@@ -0,0 +1,35 @@
+/*global define*/
+define([
+        './freezeObject'
+    ], function(
+        freezeObject) {
+    "use strict";
+
+    /**
+     * This enumerated type is used to determine how the vertices of the terrain mesh are compressed.
+     *
+     * @namespace
+     * @alias TerrainCompression
+     *
+     * @private
+     */
+    var TerrainCompression = {
```

Comment:
Would it be more precise to name this `TerrainQuantization` since this not full compression?